pagespeed/envoy/BUILD

```diff
@@ -116,3 +120,116 @@ envoy_cc_test(
         "@envoy//test/integration:http_integration_lib",
     ],
 )
+
+filegroup(
+    name = "envoy_fetcher_headers",
+    srcs = [
+        "envoy_url_async_fetcher.h",
+        "envoy_cluster_manager.h",
+        "pagespeed_remote_data_fetcher.h",
+        "envoy_fetch.h",
+    ],
+)
+
+filegroup(
+    name = "envoy_cluster_config",
+    srcs = [
+        "cluster.yaml",
+    ],
+)
+
+cc_library(
+   name = "config",
+   data = [":envoy_cluster_config"],
+)
+
+envoy_cc_library(
+    name = "envoy_fetcher",
+    srcs = [
+        "envoy_url_async_fetcher.cc",
+        "envoy_cluster_manager.cc",
+        "pagespeed_remote_data_fetcher.cc",
+        "envoy_fetch.cc",
+        "header_utils.h",    
+    ],
+    hdrs = [
+        ":envoy_fetcher_headers",
+    ],
+    repository = "@envoy",
+    copts = [
+        "-Iexternal/mod_pagespeed/third_party/apr/gen/arch/linux/x64/include/",
```

Comment:
Why do we need apr?

Reply:
Removed dependency of apr. Code pushed https://github.com/apache/incubator-pagespeed-mod/pull/1969/commits/05e80e98faba31ff5a94f1244b1833fa88a1fae4